src/lib/lifecycleEvents.ts

```diff
@@ -79,7 +79,7 @@ export async function run(lifecycleEvent: string, params: GleeFunctionEvent) {
     const responses = await Promise.all(handlers.map(info => info.fn(params)))
     
     responses.forEach(res => {
-      if (res.send) {
+      if (res?.send) {
         res.send.forEach((event: GleeFunctionReturnSend) => {
```

Comment:
Wouldn't it make more sense to do this directly?

```suggestion
        res?.send?.forEach((event: GleeFunctionReturnSend) => {
```

This way we remove the `if` clause.